src/main.rs

```diff
@@ -207,7 +243,30 @@ fn main() -> Result<()> {
         }
     }
 
-    if !delete_file.exists() {
+    if cli.worktree_branch.is_some() {
+        if !delete_file.exists() {
+            println!(
+                "Working tree directory preserved at: {}",
+                tmp_dir.into_path().display()
+            );
```

Comment:
This is kinda duplicated code with the other else branch at the bottom. The only difference is the text "Project preserved" vs "Working tree directory preserved". I think this is unnecessary, the user knows very well what they started and they are both projects anyway. You can try to simplify the code by making this code common.